classes/patreon_wordpress.php

```diff
@@ -15,56 +15,58 @@ class Patreon_Wordpress {
 	private static $Patreon_Options;
 	private static $Patron_Metabox;
 	private static $Patreon_User_Profiles;
-	private static $Patreon_Shortcodes;
 
 	function __construct() {
 
 		include 'patreon_login.php';
 		include 'patreon_routing.php';
 		include 'patreon_frontend.php';
-		include 'patreon_posts.php';
 		include 'patreon_api.php';
 		include 'patreon_oauth.php';
-		include 'patreon_protect.php';
 		include 'patreon_options.php';
 		include 'patreon_metabox.php';
 		include 'patreon_user_profiles.php';
-		include 'patreon_shortcodes.php';
 
 		self::$Patreon_Routing = new Patreon_Routing;
 		self::$Patreon_Frontend = new Patreon_Frontend;
-		self::$Patreon_Posts = new Patreon_Posts;
-		self::$Patreon_Protect = new Patreon_Protect;
 		self::$Patreon_Options = new Patreon_Options;
 		self::$Patron_Metabox = new Patron_Metabox;
 		self::$Patreon_User_Profiles = new Patreon_User_Profiles;
-		self::$Patreon_Shortcodes = new Patreon_Shortcodes;
 
 		add_action('wp_head', array($this, 'updatePatreonUser') );
+		add_action('init', array($this, 'checkPatreonCreatorID'));
+		add_action('init', array($this, 'checkPatreonCreatorURL'));
+		add_action('init', array($this, 'checkPatreonCreatorName'));
+		add_action('init', 'Patreon_Login::checkTokenExpiration');
 
 	}
-
+	
 	static function getPatreonUser($user) {
 
 		/* get user meta data and query patreon api */
 		$user_meta = get_user_meta($user->ID);
 		if(isset($user_meta['patreon_access_token'][0])) {
 			$api_client = new Patreon_API($user_meta['patreon_access_token'][0]);
 
+			// Below is a code that caches user object for 60 seconds. This can be commented out depending on the response from Infrastructure team about contacting api to check for user on every page load
```

Comment:
good reminder. Let's keep this note. It might be something we have to circle back to later